src/stories/components/MainMenu/styles.scss

```diff
@@ -3,52 +3,59 @@
 // @file
 // Main Menu
 
-// .main-menu {}
+.main-menu {
+  @include list-clean;
+}
+
+.main-menu__item {
+  margin: $spacing-2 0;
+}
 
 .menu-item__link {
-	display: flex;
-    cursor: pointer;
-    font-weight:normal; 
-    text-decoration: none;
-
-    &.active {
-        .menu-item__icon {
-            background: $teal;
-            fill: $white;
-            border-top-right-radius: 100px;
-            border-bottom-right-radius: 100px;
-        }
-
-        .menu-item__text {
-            color: $teal-dark;
-        }
+  align-items: center;
+  display: flex;
+  cursor: pointer;
+  font-weight:normal;
+  text-decoration: none;
+
+  &.active {
+    .menu-item__icon {
+      background: $teal;
+      fill: $white;
     }
+
+    .menu-item__text {
+      color: $teal-dark;
+    }
+  }
 }
 
-.menu-item__link:hover {
+.menu-item__link {
+  font-family: $header-font-family;
+  font-weight: 700;
+
+  &:not(.active):hover {
```

Comment:
Neat selector which applies an `hover` style to all elements except the one that has an `active` class. Since that element is already selected, it doesn't need a hover state.